From a6ebb04f2d1db8b6096e194c91b2183120c80be4 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 13 Jun 2023 13:32:55 +0200
Subject: [PATCH 0681/1695] dts: stm32: enable HASH on stm32mp257f-ev1

Sets HASH peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git df0435128a40552ba4a5635569ac0d96a43af63a dts: stm32: enable HASH on stm32mp257f-ev1]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I81130cb77b089ed6fed43e4a87328dbcf325a2e7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/361540
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index ad630ffba..54b263ba9 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -48,6 +48,10 @@
 	};
 };
 
+&hash {
+	status = "okay";
+};
+
 &hsi_calibration {
 	status = "okay";
 };
-- 
2.47.2

